madmom/audio/chroma.py

```diff
@@ -247,3 +249,100 @@ def __init__(self, fmin=65, fmax=2100, unique_filters=True, models=None,
         super(DeepChromaProcessor, self).__init__([
             sig, frames, spec, spec_frames, _dcp_flatten, nn
         ])
+
+
+class CLPChroma(np.ndarray):
+    """
+    This class implements Compressed Log Pitch (CLP) chroma as proposed in [
+    1] and [2].
+
+    Parameters
+    ----------
+    data : str, or TimeDomainSemitoneFilteredSpectrogram
+        Semitone (MIDI notes) spectrogram or file name.
+    fps : int, optional
+        Desired sample rate of the signal [Hz].
+    midi_min : int, optional
```

Comment:
I'd prefer to have `fmin` and `fmax` given in Hz instead of MIDI note numbers (as used throughout this module).